R/generate-ngm.R

```diff
@@ -0,0 +1,79 @@
+#' Get next generation contact matrices
+#' @param state_name target state name
+#' @param lga_name target LGA name
+#' @param age_breaks age breaks
+#' @param R_target target reproduction number
+#'
+#' @export
+#' @examples
+#' \dontrun{
```

Comment:
If this takes longer than 1 minute to run we can leave the `\dontrun` in there and just note how long it generally takes to run and give that reason for skipping it. 

Otherwise it would be good to remove `\dontrun`